src/ArduinoIoTCloudTCP.cpp

```diff
@@ -581,6 +580,30 @@ void ArduinoIoTCloudTCP::sendPropertiesToCloud()
     }
 }
 
+void ArduinoIoTCloudTCP::sendPropertiesToCloud()
+{
+  sendPropertyContainerToCloud(_property_container);
+}
+
+void ArduinoIoTCloudTCP::sendOTAPropertiesToCloud()
+{
+  PropertyContainer to_property_container;
+  PropertyContainer from_property_container = _property_container;
+
+  std::list<String> ota_property_list {"OTA_CAP", "OTA_ERROR", "OTA_SHA256", "OTA_URL", "OTA_REQ"};
```

Comment:
Minor nitpick: `ota_property_list` can be const, which does entail using `cbegin()`, `cend()` in `std::for_each` below, i.e.
```C++
std::list<String> const ota_property_list {"OTA_CAP", "OTA_ERROR", "OTA_SHA256", "OTA_URL", "OTA_REQ"};
  std::for_each(ota_property_list.cbegin(),
                ota_property_list.cend(),
/* ... */
```